### ISSUE TITLE
Should Kit files auto re-compile when referenced partials change (currently they are not)

### ISSUE THREAD

**Issue by benfrain** [bdkjones/CodeKit-1-Old#501]
Just giving Kit a whirl and have a basic conceptual question...

Say I have a kit partial for my navigation section (_nav.kit) and I reference/include a link to that file in all my other files (e.g. index.kit contact.kit).

When I make a change to nav.kit - should the HTML files that are generated from index.hit (index.html) and contact.kit (contact.html) get re-compiled with the relevant changes? That's not happening at present.

I currently have to manually re-open any file referencing nav.kit and hit save to get it re-made with the updates. Is that what is supposed to happen?

**Comment by bdkjones** [comment]
Please send me a sample project that demos the issue so I can take a look. Thanks!

**Comment by benfrain** [comment]
Hi Bryan, cheers. No problem sending a demo project if this seems like a bug. However, it could just be a disparity between my expectations and the way it actually works. Is the above the intended behaviour?

**Comment by bdkjones** [comment]
No. When you save a .kit file that is imported into others, the master files should recompile to capture the changes (just as they do in all other languages CodeKit supports).

**Comment by benfrain** [comment]
Brian, figured the issue. Realised it needed to be .kit files into .kit files for it to work. Was trying to do .html into .kit and wondering why changes of the .html files weren't making the 'master' kit file recompile. Put this one down to 'error between keyboard and chair' ;)

**Comment by bdkjones** [comment]
Actually, I want CodeKit to recompile the master Kit file when _any_ imported file changes, so re-opened and marked for fixing.
